pkg/oauth/api/register.go

```diff
@@ -37,11 +37,16 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 		&OAuthClientList{},
 		&OAuthClientAuthorization{},
 		&OAuthClientAuthorizationList{},
+		&SelfOAuthClientAuthorization{},
+		&SelfOAuthClientAuthorizationList{},
 		&OAuthRedirectReference{},
 	)
 	return nil
 }
 
+func (obj *SelfOAuthClientAuthorizationList) GetObjectKind() unversioned.ObjectKind {
+	return &obj.TypeMeta
```

Comment:
Impossible, I have it on by default on each save and it just works as expected with those.